docs/install/install.sh.md

```diff
@@ -27,87 +27,6 @@ manually via `coder server` or as a system package.
 By default, the Coder server runs on `http://127.0.0.1:3000` and uses a
 [public tunnel](../admin/configure.md#tunnel) for workspace connections.
 
-## PATH conflicts
-
-It's possible to end up in situations where you have multiple `coder` binaries
-in your `PATH`, and your system may use a version that you don't intend. Your
-`PATH` is a variable that tells your shell where to look for programs to run.
-
-You can check where all of the versions are by running `which -a coder`.
-
-For example, a common conflict on macOS might be between a version installed by
-Homebrew, and a version installed manually to the /usr/local/bin directory.
-
-```console
-$ which -a coder
-/usr/local/bin/coder
-/opt/homebrew/bin/coder
-```
-
-Whichever binary comes first in this list will be used when running `coder`
-commands.
-
-### Reordering your PATH
-
-If you use bash or zsh, you can update your `PATH` like this:
-
-```shell
-# You might want to add this line to the end of your ~/.bashrc or ~/.zshrc file!
-export PATH="/opt/homebrew/bin:$PATH"
-```
-
-If you use fish, you can update your `PATH` like this:
-
-```shell
-# You might want to add this line to the end of your ~/.config/fish/config.fish file!
-fish_add_path "/opt/homebrew/bin"
-```
```

Comment:
Our install script now defaults to homebrew for macOS, so this issue will likely not exist anymore. Still, I am moving this to the FAQ section.